app/controllers/WhatDoYouWantToDoController.scala

```diff
@@ -18,25 +18,23 @@ package controllers
 
```

Comment:
Do these changes need to be done in the backend aswell?

Reply:
Yes but for backend we may need to agree on a convention. Fix the responses returned so that they can be interpreted accordingly by frontend